docs/guides/orms.mdx

```diff
@@ -144,7 +144,7 @@ pgroll convert 0000_init.sql
 
 ## Limitations
 
-The functionality still has some limitations. The generated pgroll migrations must be edited to provide `up` and `down` data migrations manually. You can find more details about writing these migrations in [this guide](/updown).
+The functionality still has some limitations. The generated pgroll migrations must be edited to provide `up` and `down` data migrations manually. You can find more details about writing these migrations in [this guide](updown).
```

Comment:
`/updown` was 404ing. Should be fine without the leading `/` since `updown.mdx` is in the same dir as `orms.mdx`.